compass/ocean/tests/global_ocean/init/initial_state.py

```diff
@@ -160,6 +170,36 @@ def run(self):
 
         run_model(self)
 
+        if self.with_inactive_top_cells:
+
+            logger.info("   * Updating minLevelCell for inactive top cells")
+
+            in_filename = 'initial_state.nc'
+            out_filename = in_filename
+
+            with xarray.open_dataset(in_filename) as ds:
+
+                # keep the data set with Time for output
+                ds_out = ds
+
+                ds = ds.isel(Time=0)
+
+                if ('minLevelCell' in ds):
```

Comment:
```suggestion
                if 'minLevelCell' in ds:
```
These parentheses aren't needed.  It doesn't seem to trigger a PEP8 warning but I think we want to avoid them in the standard compass coding style nonetheless.

There are some other PEP8 warnings about trailing white space that it would be good to have fixed as well.